samples/map/singleStepMap/main.cpp

```diff
@@ -0,0 +1,74 @@
+/**
+* @version		GrPPI v0.1
+* @copyright		Copyright (C) 2017 Universidad Carlos III de Madrid. All rights reserved.
+* @license		GNU/GPL, see LICENSE.txt
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You have received a copy of the GNU General Public License in LICENSE.txt
+* also available in <http://www.gnu.org/licenses/gpl.html>.
+*
+* See COPYRIGHT.txt for copyright notices and details.
+*/
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <chrono>
+#include <string>
+#include "map.h"
+
+using namespace grppi;
+
```

Comment:
Violates coding guideline.

You can use a  using namspace grppi, inside individual functions, but not at global scope.